modules/network-observability-working-with-conversations.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-working-with-conversations_{context}"]
+= Working with conversation tracking
+As an administrator, you can you can group network flows that are part of the same conversation. A conversation is defined as a grouping of peers that are identified by their IP addresses, ports, and protocols, resulting in an unique *Conversation Id*. You can query conversation events in the web console. These events are represented in the web console as follows:
+
+** *Conversation start* happens when a connection is starting or TCP flag intercepted
```

Comment:
I might update the list like:

*Conversation start*. This event happens

I think separating the term from the definition is the more common method in our docs.

Reply:
s/TCP flag intercepted/TCP flag is intercepted ??

Reply:
Good suggestion. I'll take it :)